aws-sns-topicinlinepolicy/src/main/java/software/amazon/sns/topicinlinepolicy/DeleteHandler.java

```diff
@@ -0,0 +1,75 @@
+package software.amazon.sns.topicinlinepolicy;
+
+import com.amazonaws.util.StringUtils;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.SetTopicAttributesResponse;
+import software.amazon.awssdk.utils.CollectionUtils;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.HandlerErrorCode;
+
+public class DeleteHandler extends BaseHandlerStd {
+    private software.amazon.cloudformation.proxy.Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<SnsClient> proxyClient,
+            final Logger logger) {
+
+        this.logger = logger;
+        ResourceModel resourceModel = request.getDesiredResourceState();
+
+        if (resourceModel == null || StringUtils.isNullOrEmpty(resourceModel.getTopicArn())) {
+            return ProgressEvent.failed(resourceModel, callbackContext, HandlerErrorCode.InvalidRequest, "TopicArn can not be empty" );
```

Comment:
How about making `TopicArn can not be empty` a static string, and refer in both code and UT?